src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebReportRouteConfig.kt

```diff
@@ -45,6 +45,12 @@ object WebReportRouteConfig : RouteConfig
                     .json()
                     .secure(runReports)
                     .transform(),
+            WebEndpoint(
+                    "/reports/:name/parameters/",
```

Comment:
Sorry - didn't spot this first time, but for consistency this should be `/report/`, not `/reports/`.